astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Startup.cs"}
+  lang="c#" />
+
+<Aside type="note">
+There are a few settings applied to simplify the sample application. In a production application, do not allow cookies over `http` or keep the `ClientSecret` in a .json file for production applications.  Get the `ClientSecret` from an environmental variable that has been set on a production machine or some other secure way.
+</Aside>
+
+#### Navigation
+This application will display different menu items for authenticated users and non authenticated users. To add this type of navigation update `/Pages/Shared/_Layout.cshtml` with the following:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Shared/_Layout.cshtml"}
+  lang="razor" />
+
+#### Homepage
+Update the default page.
+
+Copy the following code into `Index.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml.cs"}
+  lang="c#" />
+
+Copy the following code into `Index.cshtml` and notice the logic to show a different view to users that have not logged in.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml"}
+  lang="razor" />
+
+#### Styling
+To give the pages the look and feel of an application, apply the following styling and images.
+
+Add the `changebank.css` file in the `wwwroot\css" directory of the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/wwwroot/css/changebank.css"}
+  lang="css" />
+
+Add the Changebank logo to the application.
+
+Copy the money image for the `Index` page.
+
+```shell
+curl --create-dirs --output wwwroot/img/money.jpg  https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-web/main/complete-app/app/assets/images/money.jpg
+```
+
+## Conclusion
+
+Your directory for the `complete-application` tree should look like this:
+
+```text
+└── complete-application
+    ├── appsettings.Development.json
+    ├── appsettings.json
+    ├── bin/
+        ├── ...
+    ├── obj/
+        ├── ...
+    ├── Pages/
+    │   ├── _ViewImports.cshtml
+    │   ├── _ViewStart.cshtml
+    |   |── Account.cshtml
+    |   |── Accuont.cshtml.cs
+    │   ├── Error.cshtml
+    │   ├── Error.cshtml.cs
+    │   ├── Index.cshtml
+    │   ├── Index.cshtml.cs
+    │   ├── Logout.cshtml
+    │   ├── Logout.cshtml.cs
+    |   |── MakeChange.cshtml
+    |   |── MakeChange.cshtml.cs
+    │   ├── Privacy.cshtml
+    │   ├── Privacy.cshtml.cs
+    │   └── Shared/
+    │       ├── _Layout.cshtml
+    │       ├── _Layout.cshtml.css
+    │       └── _ValidateScriptsPartial.cshtml
+    ├── Program.cs
+    ├── Properties/
+        ├── ...
+    ├── Startup.cs
+    └── wwwroot/
+        ├── ...
+```
+
+Once you've created this directory structure, you can start up the {frontmatter.technology} application using this command:
```

Comment:
Let's create a new section called 'Run the App'. and move the conclusion below this, please.

Reply:
done